lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2522,6 +2522,18 @@ namespace {
           VarDecl *pointeeProperty =
               synthesizer.makeDereferencedPointeeProperty(
                   getterAndSetter.first, getterAndSetter.second);
+
+          // Import the attributes from dereference operator clang decl to
+          // pointee property swift decl.
+          FuncDecl *setterImpl = getterAndSetter.second;
```

Comment:
Does the code use `setterImpl` anywhere? Looks like this is an unused assignment.

Reply:
ohh yes, this is useless. I should remove this variable!